.github/workflows/suite.yml

```diff
@@ -117,43 +131,48 @@ jobs:
       matrix:
         workload: ${{ fromJSON(github.event.inputs.network_workloads) }}
     name: ${{ matrix.workload }} (network)
-    runs-on: ["runs-on", "runner=16cpu-linux-x64", "spot=false", "run-id=${{ github.run_id }}"]
+    runs-on:
+      [
+        "runs-on",
+        "runner=16cpu-linux-x64",
+        "spot=false",
+        "run-id=${{ github.run_id }}",
+      ]
     steps:
-    - name: Checkout sources
-      uses: actions/checkout@v4
-
-    - name: Setup CI
-      uses: ./.github/actions/setup
-
-    - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1
-      with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_S3 }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_S3 }}
-        aws-region: us-west-2
-
-    - name: Copy files from S3
-      run: |
-        mkdir -p workdir
-        aws s3 cp s3://sp1-testing-suite/${{ matrix.workload }}/program.bin workdir/program.bin
-        aws s3 cp s3://sp1-testing-suite/${{ matrix.workload }}/stdin.bin workdir/stdin.bin
-
-    - name: Clean Cargo cache
-      uses: actions-rs/cargo@v1
-      with:
-        command: clean
-        toolchain: 1.81.0
-
-    - name: Run sp1-perf
-      uses: actions-rs/cargo@v1
-      with:
-        command: run
-        toolchain: 1.81.0
-        args: --release -p sp1-perf --features native-gnark -- --program workdir/program.bin --stdin workdir/stdin.bin --mode network
-      env:
-        RUST_LOG: info
-        RUSTFLAGS: -Copt-level=3 -Ctarget-cpu=native
-        RUST_BACKTRACE: 1
-        SP1_PROVER: network
-        SP1_PRIVATE_KEY: ${{ secrets.SP1_PRIVATE_KEY }}
-        PROVER_NETWORK_RPC: https://rpc-staging.succinct.xyz
\ No newline at end of file
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Setup CI
+        uses: ./.github/actions/setup
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_S3 }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_S3 }}
+          aws-region: us-west-2
+
+      - name: Copy files from S3
+        run: |
+          mkdir -p workdir
+          aws s3 cp s3://sp1-testing-suite/${{ matrix.workload }}/program.bin workdir/program.bin
+          aws s3 cp s3://sp1-testing-suite/${{ matrix.workload }}/stdin.bin workdir/stdin.bin
+
+      - name: Clean Cargo cache
+        uses: actions-rs/cargo@v1
+        with:
+          command: clean
+          toolchain: 1.81.0
+
+      - name: Run sp1-perf
+        uses: actions-rs/cargo@v1
+        with:
+          command: run
+          toolchain: 1.81.0
+          args: --release -p sp1-perf --features "native-gnark,network-v2" -- --program workdir/program.bin --stdin workdir/stdin.bin --mode network
```

Comment:
the downside with this approach is that we no longer test network-v1 in ci, but maybe that is fine? @jtguibas